lib/utils.js

```diff
@@ -66,6 +66,17 @@ const getAllChangedFiles = async (context) => {
   return changedFiles;
 };
 
+const checkPrIsStale = async (pullRequest, context) => {
+  const {data: lastCommit} = await context.github.repos.getCommit(
+    context.repo({
+      ref: pullRequest.head.sha
+    })
+  );
+  const lastCommitDate = new Date(lastCommit.commit.author.date);
+  // Commit is older than 2 days
+  return (MIN_BUILD_DATE > lastCommitDate);
```

Comment:
Tests are failing mainly due to the way the minimum build date is being mocked.
You should probably figure out another way to mock the date here so that it can be set in the test.

Reply:
working on it.

Reply:
@jameesjohn please can you give me a clue on how should i mock the `MIN_BUILD_DATE`.